README.md

```diff
@@ -1,4 +1,33 @@
 # oeuvres-roud
-Project specific ontologies and lists for the project "Oeuvres complètes de Gustave Roud".
 
-Salsah 1.5 User guide by @elespdn : https://github.com/gustaveroudproject/SalsahModeEmploi/blob/master/SalsahModeEmploi.md
+Project ["Oeuvres complètes de Gustave Roud"](https://www.unil.ch/clsr/home/menuinst/projets-de-recherche/gustave-roud-oeuvres-completes.html).
+
```

Comment:
Could you also add the SNSF ID of the project and the link to SNSF database? (http://p3.snf.ch/Project-157970)